addons/website/models/website_menu.py

```diff
@@ -136,6 +137,26 @@ def clean_url(self):
                     url = '/%s' % self.url
         return url
 
+    def is_active(self):
```

Comment:
Should it be `_is_active` to not expose it as a callable public method? Probably the same for `clean_url` ?

Reply:
Sure, changed

Reply:
I guess I should create a migration script for the `clean_url` changed tho, as this is typically called from templates, like in our internal repo